README.md

```diff
@@ -92,7 +92,8 @@ export default Example;
 👉 Please notice that **all the [`@react-native-community/react-native-datetimepicker`](https://github.com/react-native-community/react-native-datetimepicker) props are supported** as well!
 
 | Name                      | Type      | Default      | Description                                                                                         |
-| ------------------------- | --------- | ------------ | --------------------------------------------------------------------------------------------------- |
+|---------------------------|-----------| ------------ |-----------------------------------------------------------------------------------------------------|
+| `buttonFontColor`         | string    |              | The color of the confirm button texts (iOS)                                                         |
```

Comment:
Can we rename it to `buttonTextColorIOS`?

Reply:
@mmazzarolo Thanks for the feedback! Done! 👍